src/App.css

```diff
@@ -4,6 +4,7 @@
   padding-bottom: 3rem;
   margin: 0;
   background-color: #fafafa;
+  overflow: hidden;
 }
```

Comment:
# On 80% Screen Size
![design-on-80% screen-size](https://user-images.githubusercontent.com/93039359/187046327-74a4f4aa-0e78-4aa8-ad7d-69e958a078f6.png)

# On 100% Screen Size
![on-100%screen-size](https://user-images.githubusercontent.com/93039359/187046331-ebcb9c39-16a3-4624-970b-90d19c5ab69c.png)

- [ ] Please update your margins and paddings so you won't need to use overflow: hidden; because your design is going out of our screens which is not really professional. Please fix it by changing margins and paddings in your cards :blush:
- [ ] Your design should fit in 100% screen size :thinking: